### ISSUE TITLE
Error: Something went wrong creating the customized pack: Error occurred while fetching all worldgen/configured_feature presets: Failed to fetch

### ISSUE THREAD

**Issue by lantern-ghost** [misode/misode.github.io#394]
## Crash report
 * Page url: `https://misode.github.io/customized/?tab=basic`
 * Current version: `1.20`
 * Latest version: `1.20.1`

### Stack trace
```
Error: Something went wrong creating the customized pack: Error occurred while fetching all worldgen/configured_feature presets: Failed to fetch
    at Oa (src/app/services/DataFetcher.ts:133:8)
    at async Promise.all (index 6)
    at async (src/app/components/customized/CustomizedGenerator.ts:23:40)
    at async (src/app/pages/Customized.tsx:60:16)
```

### Customized settings
<details>
<pre>
{}
</pre>
</details>

**Comment by misode** [comment]
Possibly related to #360, but I thought this was originally fixed by #239.

@lantern-ghost I added some additional debugging messages, could you try again and check the error message?

Additionally can you try opening these URLS to see whether they are blocked in your country?
- https://raw.githubusercontent.com/misode/mcmeta/1.20.1-summary/data/worldgen/configured_feature/data.min.json
- https://cdn.jsdelivr.net/gh/misode/mcmeta@1.20.1-summary/data/worldgen/configured_feature/data.min.json

**Comment by misode** [comment]
Marking as duplicate of #360
